docs/clients/snappi.md

```diff
@@ -0,0 +1,711 @@
+# snappi
```

Comment:
Somewhere on this page we need a link to [Hello snappi](https://github.com/open-traffic-generator/ixia-c/blob/main/docs/hello-snappi.md) and [Common snappi constructs](https://github.com/open-traffic-generator/ixia-c/blob/main/docs/snappi-constructs.md).

Seems that a lot of content on this page is covered there already.  Alternatively, we can make this the main page and update those to point to here.

Reply:
I don't like this rendering as well. We need to see how to make it better, but let's leave that for another set of improvements. I'll be linking "Hello snappi" from the examples section, and will link it from here as well.

Reply:
Done linking Common snappi constructs:
## Overview

Test scripts written in [`snappi`](https://github.com/open-traffic-generator/snappi), an auto-generated Python module, can be executed against any traffic generator conforming to the Open Traffic Generator API. The examples of `snappi` coding patterns can also be found as part of [Ixia-c documentation](https://github.com/open-traffic-generator/ixia-c/blob/main/docs/snappi-constructs.md).